tools/package-tools/package-version-stamp.ts

```diff
@@ -0,0 +1,30 @@
+import {writeFileSync} from 'fs';
+import {spawnSync} from 'child_process';
+import {buildConfig} from './build-config';
+
+/**
+ * Inserts the a version stamp, which consists of the current commit SHA and the current
+ * branch name, into the specified package.json file.
+ *
+ * This makes it easy to quickly verify the exact snapshot from which the release originated.
+ */
+export function insertPackageJsonVersionStamp(packageJsonPath: string) {
+  const packageJson = require(packageJsonPath);
+
+  packageJson['_gitCommitStamp'] = getCurrentCommitSha();
+  packageJson['_gitBranchStamp'] = getCurrentBranchName();
```

Comment:
I thought it might also be useful to stamp the current git username, but I want to check with Igor/Brad about doing that. I'll let you know.